source/core/src/main/com/csse3200/game/inventory/items/lootbox/configs/EarlyGameLootTable.java

```diff
@@ -0,0 +1,24 @@
+package com.csse3200.game.inventory.items.lootbox.configs;
+
+import com.csse3200.game.inventory.items.food.Foods;
+import com.csse3200.game.inventory.items.potions.AttackPotion;
+import com.csse3200.game.inventory.items.potions.DefensePotion;
+import com.csse3200.game.inventory.items.potions.HealingPotion;
+import com.csse3200.game.inventory.items.potions.SpeedPotion;
+
+public class EarlyGameLootTable extends BaseLootTable {
+
+    // Constructor to populate the loot table with default items
+    public EarlyGameLootTable() {
+
+        addItem(HealingPotion.class, 100.0, new Class<?>[]{int.class}, new Object[]{1});
```

Comment:
Maybe make a config json file for this so there aren't so many magic numbers floating around.